Automattic/wp-calypso
# Intro Banner: Hide the introductory pricing banner if running a sale

#### Changes proposed in this Pull Request

* Hides the Introductory Pricing banner if a sale is running

#### Testing instructions



_Note: we are currently running a sale, so you can go straight to the live link and check the /pricing page. If you are viewing this in the future, make sure to go through the test plan in D70003-code before applying the changes_ 
* Go to /pricing and make sure the Sale banner appears on the top of the page
* If the banner is present, the Introductory banner should be hidden. See the screenshot below

![image](https://user-images.githubusercontent.com/5714212/142918847-6e861d24-a721-4518-b78d-441f50cf760e.png)
